packages/server/lib/controllers/proxy.controller.ts

```diff
@@ -161,6 +161,31 @@ class ProxyController {
             }
             metrics.increment(metrics.Types.PROXY_FAILURE);
             next(err);
+        } finally {
+            const getHeaders = (hs: IncomingHttpHeaders | OutgoingHttpHeaders): Record<string, string> => {
+                const headers: Record<string, string> = {};
+                for (const [key, value] of Object.entries(hs)) {
+                    if (typeof value === 'string') {
+                        headers[key] = value;
+                    } else if (Array.isArray(value)) {
+                        headers[key] = value.join(', ');
+                    }
+                }
+                return headers;
+            };
+            const reqHeaders = getHeaders(req.headers);
+            reqHeaders['authorization'] = 'REDACTED';
+            await logCtx?.enrichOperation({
+                request: {
+                    url: `${req.protocol}://${req.get('host')}${req.originalUrl}`,
+                    method: req.method,
+                    headers: reqHeaders
+                },
+                response: {
+                    code: res.statusCode,
+                    headers: getHeaders(res.getHeaders())
```

Comment:
we are gonna log way more content with the headers. Is it gonna be ok with ES?

Reply:
I need to increase the disk size anyway. It will increase the storage a bit but most logs are empty (!= operation)